aw-server/src/android/mod.rs

```diff
@@ -93,33 +103,50 @@ pub mod android {
         string_to_jstring(&env, obj.to_string())
     }
 
+    lazy_static! {
+        static ref ASSET_PATH: RwLock<String> = RwLock::new(String::new());
+    }
+
     #[no_mangle]
     pub unsafe extern "C" fn Java_net_activitywatch_android_RustInterface_startServer(
         env: JNIEnv,
         _: JClass,
         java_asset_path: JString,
     ) {
-        use crate::config::AWConfig;
-        use std::path::PathBuf;
+        info!("Starting server...");
 
-        use crate::endpoints;
+        *ASSET_PATH.write().unwrap() = jstring_to_string(&env, java_asset_path);
 
-        info!("Building server state...");
+        // endpoints::build_rocket(server_state, config).launch().await;
+        start_server();
 
-        let asset_path = jstring_to_string(&env, java_asset_path);
-        info!("Using asset dir: {}", asset_path);
+        info!("Server exited");
+    }
 
-        let server_state = endpoints::ServerState {
-            datastore: Mutex::new(openDatastore()),
-            asset_path: PathBuf::from(asset_path),
-            device_id: device_id::get_device_id(),
-        };
+    #[rocket::main]
+    async fn start_server() {
+        unsafe {
```

Comment:
Why add `unsafe` to the entire block?

Reply:
I think only the first info line is safe, can commit to remove the first line, just threw all in together for now